OpenOversight/app/gmail_client.py

```diff
@@ -0,0 +1,105 @@
+"""This file houses the classes used to send emails."""
+import base64
+from email.mime.text import MIMEText
+
+from apiclient import errors
+from flask import current_app, render_template
+from google.oauth2 import service_account
+from googleapiclient.discovery import build
+
+
+DEFAULT_SENDER_ADDRESS = "chi-oo@lucyparsonslabs.com"
+
+
+class Email:
```

Comment:
I'm hoping this inherited object structure will give us some flexibility when it comes to multiple email types.